apps/condo/lang/en/en.json

```diff
@@ -497,6 +503,11 @@
   "pages.condo.property.warning.modal.Title": "Do not save property?",
   "pages.condo.property.id.DownloadExcelLabel": "Download file",
   "pages.condo.property.ImportTitle": "properties",
+  "pages.condo.property.id.PropertyInformationTitle": "About property",
+  "pages.condo.property.id.PropertyReportTitle": "Property report",
+  "pages.condo.property.id.PropertyReportDescription": "If you are Sber client, you're able to show cool report to your residents. If not a client - become one by leaving a request",
```

Comment:
```suggestion
  "pages.condo.property.id.PropertyReportDescription": "If You are a client of Sber, You're able to present a cool report of property expenses to Your residents. Otherwise, become one by sending a request",
```